xcm/pallet-xcm/src/lib.rs

```diff
@@ -232,6 +232,9 @@ pub mod pallet {
 		/// `pallet_xcm::CurrentXcmVersion`.
 		type AdvertisedXcmVersion: Get<XcmVersion>;
 
+		/// The configurable origin to use with Gov2
```

Comment:
This is not actually what the `AdminOrigin` is supposed to do. It's an origin that is allowed to call privileged operations on the XCM pallet.

Reply:
fixed the comment line.

Reply:
For my own understanding, is it correct that, in future, `AdminOrigin` can include more origins and not only root?

Reply:
Correct, that's the entire purpose of the issue.